packages/code/src/Code.tsx

```diff
@@ -140,8 +282,34 @@ function Code({
   chromeTitle = '',
   ...rest
 }: CodeProps) {
+  const scrollableMultiLine = useRef<HTMLPreElement>(null);
+  const scrollableSingleLine = useRef<HTMLDivElement>(null);
+  const [state, dispatch] = useReducer(scrollReducer, initialState);
+  useLayoutEffect(() => {
```

Comment:
nit, line break to break up variable declarations and the hook

Reply:
We only want this running on first run of the component?

`useLayoutEffect` has the same API as `useEffect` just differs in when the callback is called.

That being said if you go to [the docs page for useEffect](https://reactjs.org/docs/hooks-reference.html#useeffect) and search for the text "If you want to run an effect and clean it up only once" to see how we can have this hook only run once.